Modules/Core/Common/include/itkResourceProbe.h

```diff
@@ -179,6 +180,10 @@ class ITK_TEMPLATE_EXPORT ResourceProbe
   /** Obsolete member function from ITK 4.8 - 4.13. Does not do anything anymore. */
   itkLegacyMacro(virtual void GetSystemInformation());
 
+  /** Cause the object to print itself out. */
+  virtual void
+  Print(std::ostream & os, Indent indent = 0) const;
```

Comment:
Are you sure you want a default agument `= 0` here? 

In general, in C++, default arguments for parameters of virtual member functions are not recommended. There is even a Clang-Tidy warning to avoid them: https://clang.llvm.org/extra/clang-tidy/checks/google/default-arguments.html

https://google.github.io/styleguide/cppguide.html#Default_Arguments says:
> Default arguments are banned on virtual functions, where they don't work properly

Reply:
:+1: Removed. There are 3 other instances that have a default 0 as well; you may want to propose a separate PR if they definitely should not have it. And `itk::LightObject`'s method is not marked as `virtual` either.

Reply:
Thanks Jon. By the way, `LightObject::Print` is non-virtual, and I think that's OK. So for  `LightObject::Print`, it also looks OK to me to have a default argument (`= 0`).